File: problemas/ad-hoc/2377/2377.lua
numeros = {}
for numero in io.read():gmatch("%S+") do
    numeros[#numeros + 1] = tonumber(numero)
end
l = numeros[1]
d = numeros[2]

numeros = {}
for numero in io.read():gmatch("%S+") do
    numeros[#numeros + 1] = tonumber(numero)
end
k = numeros[1]
p = numeros[2]

print(math.floor(l / d) * p + l * k)

File: problemas/ad-hoc/2377/2377.clj
(require '[clojure.string :as str])

(let [[l d] (map read-string (str/split (read-line) #" "))
      [k p] (map read-string (str/split (read-line) #" "))]
  (printf "%d\n" (+ (* k l) (* p (int (Math/floor (/ l d)))))))